src/validation/nationality.validation.ts

```diff
@@ -86,7 +85,7 @@ export const validateNationality2 = (nationality: string[], nationalityValidatio
   const nationalityList = NATIONALITY_LIST.split(";");
   if (nationality[1]) {
     if (validationError.every(error => error.link !== DirectorField.NATIONALITY_2)) {
-      if (!nationality[1].trim().length || invalidPattern(nationality[1], VALID_NATIONALITY_CHARACTER)) {
+      if (!nationality[1].trim().length || !isNationalityInList(nationality[1], nationalityList)) {
           validationError.push(nationalityValidationType.Nationality2InvalidCharacter.ErrorField);
       } else if ((nationality[1].length > 50)) {
```

Comment:
For this bug, we maybe could check:
if (!nationality[1].trim().length || (invalidPattern(nationality[1], VALID_NATIONALITY_CHARACTER) && !isNationalityInList(nationality[1], nationalityList))) {
}
So only say invalid pattern if regex fails and not on list.

Reply:
We've also confirmed that countries in list won't trip up on char validation and in practice it will be country not being on list that will catch errors with char validation just an extra defence.